arcgis-ios-sdk-samples/Utility network/Configure subnetwork trace/ConfigureSubnetworkTraceOptionsViewController.swift

```diff
@@ -142,7 +142,7 @@ class ConfigureSubnetworkTraceOptionsViewController: UITableViewController {
     func showAttributePicker() {
         let selectedIndex = possibleAttributes.firstIndex { $0 == selectedAttribute }
         let optionsViewController = OptionsTableViewController(labels: possibleAttributes.map { $0.name }, selectedIndex: selectedIndex) { newIndex in
-            self.selectedAttribute = self.possibleAttributes[newIndex]
+            self.selectedAttribute = self.possibleAttributes[newIndex!!]
```

Comment:
There is an extra trailing bang ❗  that gives a build error.

Reply:
Oops, forgot to push my last commit. It's fixed now!